python/lsst/ap/association/loadDiaCatalogs.py

```diff
@@ -93,24 +144,24 @@ def run(self, exposure, apdb, doLoadForcedSources=True):
         RuntimeError
             Raised if the Database query failed to load DiaObjects.
         """
-        region = self._getRegion(exposure)
-        schema = readSchemaFromApdb(apdb)
+        baseRegion = regionTime.region
+        region = baseRegion.getBoundingCircle()
+        region.dilateBy(lsst.sphgeom.Angle(np.deg2rad(self.config.angleMargin/3600.)))
```

Comment:
It's a shame we can't use `geom.AngleUnit`, but this can still be simplified a bit:
```suggestion
        region.dilateBy(lsst.sphgeom.Angle.fromDegrees(angleMargin/3600.))
```

Reply:
Thanks for the catch, that allows us to not have to import numpy.